### PR TITLE
Feat: Alert component

### PR DESCRIPTION
**Pull request by dikshadhote**
Added following types of alert 👍 
1.Primary-alert
2.Secondary-alert
3.Success-alert
4.Warning-alert
5.Error-alert

Deployed link : https://dream-ui-alert.netlify.app/documents/documents.html
image

image

### CODE REVIEW

File: documents/documents.html [hunk 1]
```diff
@@ -98,26 +99,87 @@
...
+          <h2>Installation Guide</h2>
+          <p class="mt-1 mb-1">
+            To use dreamUI in your project,copy the link given below and paste
+            it in the "head" tag of your html,above all other style sheets.
+          </p>
+          <div class="mt-2 code-snippet">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%2821%2C23%2C24%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522dreamUi%2520lnk%2522%253E%2520%2520%2520%2520%2520%250A"
+              style="
+                width: 518px;
```
Comment (dev-enforced) [inline documents/documents.html:112]: The inline styles which you have mentioned over here you can create a utility class for it depending on the purpose.

File: base.css [hunk 2]
```diff
@@ -12,7 +12,13 @@
     --white-page-background-color:#ffffffeb;
     --text-black-color:#333333;
     --text-white-color:#ffffffeb;
+    --text-grey-color:#605d5d;
     --grey-border-color:#9790908a;
+    --component-primary-color:#9edff5;
```
Comment (abhijitdotsharma) [inline base.css:17]: A suggestion to change --component-primary-color to --primary-color, If you haven't used --primary-color in other places.

**Review by dev-enforced — APPROVED** [review]
The code looks really good and the use of variables for the appropriate conventions is good!!
You can deploy this on Vercel or Netlify so that we can view the output and point out any mistakes if any.

File: base.css (hunk 2 above)
Comment (dikshadhote) [reply base.css:17]: Hi Abhijit,
          I am using primary color class for components library's theme color. The component-primary-color is used for blue which can be used by user as there primary color.so, can not change it to --primary-color.
Any other suggestions?

File: documents/documents.html (hunk 1 above)
Comment (dikshadhote) [reply documents/documents.html:112]: The inline style came with iframe from carbon UI. Do I need to change it? May be it could have some consequences

**Comment by dikshadhote** [comment]
> The code looks really good and the use of variables for the appropriate conventions is good!! You can deploy this on Vercel or Netlify so that we can view the output and point out any mistakes if any.

https://dream-ui-alert.netlify.app/documents/documents.html
here is the deployed link. I have to fix the sidebar menu, you can review changes for alert

**Review by Raghav888 — APPROVED** [review]

File: base.css (hunk 2 above)
Comment (abhijitdotsharma) [reply base.css:17]: Okay, I had read that we should use 3word classes less, If you are already using primary-color class then this is fine.

**Review by abhijitdotsharma — COMMENTED** [review]
Looks great.

**Pull request merged**